src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.temp.Figure;
+
+public class Circle extends Figure {
+    private double radius;
+    private double figureArea;
```

Comment:
```suggestion
public class Circle extends Figure {
    private double radius;
```

нам не потрібне поле для площі, так як ми рахуємо її динамічно за допомогою методу getArea() (який має рахувати площу і повертати її значення в якості вихідного параметру)